.github/workflows/scan.yml

```diff
@@ -76,3 +76,5 @@ jobs:
           args: >
             -Dsonar.organization=${{ github.repository_owner }}
             -Dsonar.projectKey=${{ github.repository_owner }}_${{ github.event.repository.name }}
+            -Dsonar.sources=src/
+            -Dsonar.tests=src/test/,examples/spring-boot-3-jdk-17/
```

Comment:
❓ Will including the example produce valuable results? Should "examples" be wildcarded?

Reply:
I think it's worth including the examples just in case something makes its way into the code that shouldn't be there, (e.g., a token or some other sensitive artifact) that might go overlooked during a review. Would be good to have some visibility. Do you think examples belong in `sonar.sources` instead of `sonar-tests` or were you wanting to exclude the examples?

Reply:
Feels to me like a wildcard of examples should be in sources, not tests.

Reply:
@withinfocus are wildcards allowed? I ask because the SonarCloud docs state, "Wildcards (*, ** and ?) are not allowed." under `sonar.sources` in [analysis-scope](https://docs.sonarsource.com/sonarcloud/advanced-setup/analysis-parameters/)

Reply:
I could have been more specific -- it's the lack of anything after the slash that covers this, so I believe `examples/` would work.

Reply:
Done